docs/deployment/pull-backup.rst

```diff
@@ -0,0 +1,116 @@
+    need to trust the backup server (data is encrypted). In pull mode the server
+    (when logged in as root) could cause unlimited damage to the client. Therefore,
+    pull mode should be used only from servers you do fully trust!
+
+Creating a backup
+-----------------
+
+In this approach the client file system is simply mounted and then backed up.
+Note that the backup is created from within the mount point so that all files
+in the archive have their original paths (otherwise they would be backed up with
+the mount point prefix, e.g. /mnt/sshfs/bin/bash instead of /bin/bash).
+
+::
+
+    # Mount client root file system.
+    mkdir /mnt/sshfs
+    sshfs root@host:/ /mnt/sshfs
+    # Change into mount dir and back up.
+    cd /mnt/sshfs
+    borg create /path/to/repo::archive .
+    # Unmount client.
+    cd ~
+    umount /mnt/sshfs
+
+Restoring a full backup
+-----------------------
+
+In case of a restore there is no direct way for borg to know the correct UID:GID
+of files, because it uses the ``/etc/passwd`` and ``/etc/group`` of the backup
+server. Using the archived UIDs/GIDs might lead to wrong results, because after
+a reinstall the name-to-ID mapping might have changed.
+
+The workaround is chrooting into an sshfs mounted directory. In this example the
+whole client root file system is mounted and restored. We use the stand-alone
+BorgBackup executable and copy it into the mounted file system to make borg
+available after entering chroot; this can be skipped of BorgBackup is already
+installed on the client.
+
+::
+
+    # Mount client root file system.
+    mkdir /tmp/sshfs
+    sshfs root@host:/ /tmp/sshfs
+    # Mount BorgBackup repository inside it.
+    mkdir /tmp/sshfs/borgrepo
+    mount --bind /path/to/repo /tmp/sshfs/borgrepo
+    # Make borg executable available.
+    cp /usr/local/bin/borg /tmp/sshfs/usr/local/bin/borg
+    # Mount important system directories and enter chroot.
+    cd /tmp/sshfs
+    for i in dev proc sys; do mount --bind /$i $i; done
+    chroot /tmp/sshfs
+
+Now we are on the backup system but inside a chroot with the target's root file
+system. We have a copy of BorgBackup binary in ``/usr/local/bin`` and the
+repository in ``/borgrepo``, and can run a simple
+
+::
+
+    borg extract /borgrepo::archive
+
+to restore the whole archive. Finally we need to exit chroot and unmount all the
+stuff:
+
+::
+
+    exit # exit chroot
+    cd /tmp/sshfs
+    for i in dev proc sys borgrepo; do umount ./$i; done
+    cd ~
+    umount /tmp/sshfs
+
+Thanks to secuser on IRC for this howto.
+
+Simple full restore
+-------------------
+
+If you don't care about the correct UIDs/GIDs, or you can guarantee that they
+have not changed, a simple full restore can be achieved with
+
+::
+
+    borg export-tar /path/to/repo::archive | ssh root@host 'tar xf -'
+
+(let Borg export the archive in tar format, upload it to the client and directly
+extract it in root file system).
```

Comment:
guess it would be extracted into root's home directory as that would be the cwd when invoking tar?

Reply:
Right, it should be / - I'll fix that.